NEL.MESH/Services/Foundations/Mesh/MeshService.Validation.cs

```diff
@@ -56,6 +57,7 @@ private static void ValidateMeshMessageOnSendMessage(Message message, string aut
                 (Rule: IsInvalid(message.Headers, "Mex-From"), Parameter: "Mex-From"),
                 (Rule: IsInvalid(message.Headers, "Mex-To"), Parameter: "Mex-To"),
                 (Rule: IsInvalid(message.Headers, "Mex-WorkflowID"), Parameter: "Mex-WorkflowID"),
+                (Rule: IsInvalid(message.Headers, "Mex-Chunk-Range"), Parameter: "Mex-Chunk-Range"),
```

Comment:
You can't add this here, this header is optional.   If you have something that depend on this, move it into its own validation check at the point where it is required.
https://digital.nhs.uk/developer/api-catalogue/message-exchange-for-social-care-and-health-api#post-/messageexchange/-mailbox_id-/outbox